eventorder_test.go

```diff
@@ -100,6 +100,7 @@ func TestEventOrderGuarantee(t *testing.T) {
 
 	t.Logf("Starting rudder-server")
 	t.Setenv("DEPLOYMENT_TYPE", string(deployment.DedicatedType))
+	t.Setenv("RSERVER_MODE", "normal")
```

Comment:
Why did we have to set this env explicitly?  Isn't it in normal mode by default for Dedicated Deployment

Reply:
At some point while running tests and having the server panic, I think the server could no longer start in normal mode, that's why I had to fix it

Reply:
A better way would be to set the `recovery.storagePath` and clean up the directory every time?